src/Helpers/MiscHelper.js

```diff
@@ -13,6 +13,14 @@ export const dataShape = propTypes.shape({
     isLoading: propTypes.bool
 });
 
+/**
+ * Based on the  allowedParams it will contruct the arguments for the API call
```

Comment:
There is one type here and on the next  line there is some confusing sentence ( vulnerabilities-client is written is typescript). These are tiny things ::)

Reply:
In order to perform the HTTP request to the backend, we use this repo https://github.com/RedHatInsights/javascript-clients

More specifically the `vulnerabilities-client` is this: https://github.com/RedHatInsights/javascript-clients/tree/master/packages/vulnerabilities 

So for example if you want to fetch the [systemList](https://github.com/RedHatInsights/javascript-clients/blob/master/packages/vulnerabilities/git-api/api.ts#L1916) you need to pass ALL THESE parameters. 
This is what `constructParameters` is doing. 

Is it more clear ?

Reply:
I understand now. I was confused reading 'vulnerabilities is written is 'vulnerabilities-client is written is typescript'